apps/server/lib/lexical/server/code_intelligence/completion/translations/macro.ex

```diff
@@ -7,7 +7,7 @@ defmodule Lexical.Server.CodeIntelligence.Completion.Translations.Macro do
   alias Lexical.Server.CodeIntelligence.Completion.Translations.Callable
   alias Lexical.Server.CodeIntelligence.Completion.Translations.Struct
 
-  @snippet_macros ~w(def defp defmacro defmacrop defimpl defmodule defprotocol defguard defguardp defexception test use)
+  @snippet_macros ~w(def defp defmacro defmacrop defimpl defmodule defprotocol defguard defguardp defexception test use ::)
```

Comment:
I don't think `::` should be considered a snippet_macros. I would prefer to create another attribute, something like:

```elixir
@unuseful_macros ~w(::)
```

By the way, I believe the completions for alias!, in, and and are not useful either. We can include them in the @By the way, I believe the completions for alias!, in, and and are not useful either. We can include them in the @unuseful_macros list as well. list as well, and at the caller side, we can do something like: `@unuseful_macros ++ @snippet_macros`

Reply:
I'd just make a function head that skips it, I think it's the only one

Reply:
Updated with the feedbacks